tensorflow/core/kernels/fused_batch_norm_op.cu.cc

```diff
@@ -50,7 +50,7 @@ __global__ void InvVarianceToVarianceKernel(int nthreads, double epsilon,
                                             int sample_size, T* variance) {
   GPU_1D_KERNEL_LOOP(index, nthreads) {
     T inv_var = variance[index];
-    T var = __fdividef(1, inv_var * inv_var) - T(epsilon);
```

Comment:
`__fdividef()` has some magic tricks in CUDA as it appears. I'd recommend using #if / #else to switch logic between CUDA and ROCm before we figure this out.